PendingReleaseNotes

```diff
@@ -24,6 +24,9 @@ Relevant tracker: https://tracker.ceph.com/issues/55521
 
   https://docs.ceph.com/en/latest/mgr/telemetry/
 
+* OSD: The issue of high CPU utilization during recovery/backfill operations
+  has been fixed. For more details, see: https://tracker.ceph.com/issues/56530.
+
```

Comment:
This should go under the 17.2.4 (release in which the bug is being fixed) section.

Reply:
Done.